docs/user-guide/02_physics/05_spectra.md

```diff
@@ -106,16 +108,26 @@ where positive and negative sums refer to \(\sigma\) and \(\pi\) lines respectiv
 $$S_I = [1, 18, 16, 1681, 2304, 729, 1936, 5490, 1936, 729, 2304, 1681, 16, 18, 1] $$
 
 # Relevant Namelist Settings
-* `calc_fida`: Calculate FIDA spectra
+* `calc_nbi`: Calculate NBI spectra
+* `calc_dcx`: Calculate DCX spectra
+* `calc_halo`: Calculate HALO spectra
```

Comment:
for the `calc_nbi`,`calc_dcx`, and `calc_halo` you need to mention that approximate spectra is calculated when `load_neutrals=1`